tests/modules/sightings/resources/test_featured_asset_guid.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
+import json
+from app.modules.sightings.models import Sighting
+from tests.modules.sightings.resources import utils as sighting_utils
+from flask import current_app
+from tests import utils
+
+
+def test_featured_asset_guid_endpoint(db, flask_app_client):
+
+    new_researcher = utils.generate_user_instance(
```

Comment:
why are you creating a new researcher here rather than using teh existing researcher_1 and reseracher_2